testing/xfpga/test_events_c.cpp

```diff
@@ -1066,8 +1154,295 @@ TEST_P(events_handle_p, irq_event_01) {
   ASSERT_EQ(FPGA_OK, sysfs_write_u64(path.c_str(), error_csr));
   EXPECT_EQ(FPGA_OK, xfpga_fpgaUnregisterEvent(handle_accel_, FPGA_EVENT_POWER_THERMAL, eh_));
 }
-
-//INSTANTIATE_TEST_CASE_P(events, events_handle_p,
-//                        ::testing::ValuesIn(test_platform::keys()));
 INSTANTIATE_TEST_CASE_P(events, events_handle_p,
                         ::testing::Values("skx-p-1s"));
+
+class mock_events_handle_p : public testing::TestWithParam<std::string> {
+ protected:
+  mock_events_handle_p()
+      : tokens_accel_{{nullptr, nullptr}},
+        tokens_dev_{{nullptr, nullptr}},
+        tmpfpgad_log_("tmpfpgad-XXXXXX.log"),
+        tmpfpgad_pid_("tmpfpgad-XXXXXX.pid"),
+        handle_accel_(nullptr),
+        handle_dev_(nullptr) {}
+
+  virtual void SetUp() override {
+    tmpfpgad_log_ = mkstemp(const_cast<char *>(tmpfpgad_log_.c_str()));
+    tmpfpgad_pid_ = mkstemp(const_cast<char *>(tmpfpgad_pid_.c_str()));
```

Comment:
These two lines are incorrect.  mkstemp actually returns an open file descriptor (int).  See testing/fpgad/test_evt_c.cpp for an example.  I think we still have some instances of this to scrub from other tests as well.

Reply:
Will fix those.